src/coins.h

```diff
@@ -252,6 +254,14 @@ class CCoins
                 return false;
         return true;
     }
+
+    size_t DynamicMemoryUsage() const {
+        size_t ret = memusage::DynamicUsage(vout);
+        BOOST_FOREACH(const CTxOut &out, vout) {
```

Comment:
It's better to use native c++11 foreach, cause we add another dependency with the BOOST_FOREACH. Bitcoin developers have already changed Boost features to c++11 features.

Reply:
I've heard of some issues with c++11 on Dash bases. I could be mistaken here. Currently im just referencing some of bitcoin's older commits from before they had switched to c++11.

Reply:
Ok, no problem, if it's just a merge, it doesn't make sense to edit it. Anyway, there should not be any troubles with c++11, it's a standard for a long time. If foreach cycle wouldn't work, the whole C++ would be in trouble.